hr_expense_advance_clearing/wizard/account_payment_register.py

```diff
@@ -4,41 +4,73 @@
 from werkzeug.urls import url_encode
 
 from odoo import _, api, models
+from odoo.exceptions import UserError
 
 
 class AccountPaymentRegister(models.TransientModel):
     _inherit = "account.payment.register"
 
-    @api.model
-    def default_get(self, fields_list):
-        """Step to avoid UserError from core odoo:
-        - Change account type to receivable
-        - Call super() and rollback account to origin
-        """
-        advance_return = self._context.get("hr_return_advance", False)
-        origin_account = False
-        if advance_return and self._context.get("active_model") == "account.move":
+    def _get_default_advance(self):
+        """ This function is default_get from return advance only """
+        return {}
```

Comment:
```suggestion
```

Reply:
I think this is not need. Please discuss if you see why we need this.

Reply:
this function for other module need add default some field, when you return advance.

Reply:
OK.